src/client/datascience/activation.ts

```diff
@@ -51,7 +51,7 @@ export class Activation implements IExtensionSingleActivationService {
         this.PreWarmDaemonPool().ignoreErrors();
         sendTelemetryEvent(Telemetry.OpenNotebookAll);
 
-        if (!this.rawSupported.supported() && this.extensionChecker.isPythonExtensionInstalled) {
+        if (!this.rawSupported.isSupported && this.extensionChecker.isPythonExtensionInstalled) {
```

Comment:
Renamed the method to a property (found that we were still awaiting on this in some places, when it was a sync method).